modules/AM_TaskTemplates/AM_TaskTemplates_sugar.php

```diff
@@ -1,8 +1,9 @@
 <?php
-/*********************************************************************************
+/**
+ *
  * SugarCRM Community Edition is a customer relationship management program developed by
  * SugarCRM, Inc. Copyright (C) 2004-2013 SugarCRM Inc.
- *
+ * *
```

Comment:
```* *```? There seems to be quite a few of these. Can you please correct them. They also do not have the license in the coding standards.